cluster/gce/config-test.sh

```diff
@@ -95,7 +95,7 @@ ENABLE_L7_LOADBALANCING="${KUBE_ENABLE_L7_LOADBALANCING:-glbc}"
 ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 
 # Set etcd version (e.g. 3.0.4-migration.1) if you need non-default version.
-TEST_ETCD_VERSION="${TEST_ETCD_VERSION:-}"
+ETCD_VERSION="${TEST_ETCD_VERSION:-}"
```

Comment:
I did this on purpose (also to avoid changing all our test configs using this). And since this test config I think it makes sense.